test/e2e/multikueue/e2e_test.go

```diff
@@ -1126,3 +1062,23 @@ func GetMultiKueueClusterNameFromAdmissionCheckMessage(message string) string {
 	}
 	return ""
 }
+
+type objAsPtr[T any] interface {
+	client.Object
+	*T
+}
+
+func expectObjectToBeDeletedOnWorkerClusters[PtrT objAsPtr[T], T any](ctx context.Context, ns string, obj PtrT) {
+	wl := &kueue.Workload{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      workloadpytorchjob.GetWorkloadNameForPyTorchJob(obj.GetName(), obj.GetUID()),
+			Namespace: ns,
+		},
+	}
+	gomega.Eventually(func(g gomega.Gomega) {
```

Comment:
```suggestion
	gomega.Eventually(func(g gomega.Gomega) {
```
should we add offset?